packages/eslint-plugin-next/lib/rules/no-assign-module-variable.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: `Prohibit assignment to the 'module' variable`,
+      recommended: true,
+      url: 'https://nextjs.org/docs/messages/no-assign-module-variable',
+    },
+  },
+
+  create: function (context) {
+    return {
+      VariableDeclaration(node) {
+        // Checks node.declarations array for variable with id.name of 'module'
+        const moduleVariableFound = node.declarations.find(
```

Comment:
Nitpick, but semantically, using `.some` [would be better](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#:~:text=If%20you%20need%20to%20find%20if%20any%20element%20satisfies%20the%20provided%20testing%20function%2C%20use%20Array.prototype.some().) than a `.find` here, which returns a boolean.

Reply:
Agreed.